src/App.tsx

```diff
@@ -8,7 +8,69 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+import { FilterBy } from './types/Filter';
+
+const filteredTodos = (todos: Todo[], query: string, filterBy: FilterBy) => {
```

Comment:
```suggestion
const getFilteredTodos = (todos: Todo[], query: string, filterBy: FilterBy) => {
```
or `filterTodos` would be the correct name for the function
`filteredTodos` sounds like In this variable stored an array of filtered todos, not the function